src/dmd/cparse.d

```diff
@@ -2165,7 +2165,11 @@ final class CParser(AST) : Parser!AST
         out Identifier pident, StorageClass storageClass = 0)
     {
         //printf("cparseDeclarator(%d)\n", declarator);
+        AST.Types constTypes; // all the Types that will need `const` applied to them
+        constTypes.setDim(0);
 
+        AST.Type parseDecl(AST.Type t)
+        {
         AST.Type ts;
```

Comment:
There's an indentation missing here.

Reply:
I thought I'd minimize the diffs for review, and do the indent later. Ok, I added it now.

Reply:
I can hide whitespace differences in the UI, so that's not a problem.